lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -708,17 +727,20 @@ export class UserAgentApplication {
             else {
                 let logMessage;
                 if (userContainedClaims) {
-                    logMessage = "Skipped cache lookup since claims were given.";
+                    logMessage = "Skipped cache lookup since claims were given";
                 } else if (request.forceRefresh) {
                     logMessage = "Skipped cache lookup since request.forceRefresh option was set to true";
                 } else {
-                    logMessage = "Token is not in cache for scope:" + scope;
+                    logMessage = "Token is not in cache for scope: " + scope;
```

Comment:
Why is the `if` case `verbose` and `else` case `info` for logs here?

Reply:
@sameerag This is unclear. These logs were already here, we just changed the punctuation. Do you feel that info or verbose is more appropriate?

Reply:
Chose to change to verbose for now.